src/com/palmergames/util/FileMgmt.java

```diff
@@ -247,18 +246,16 @@ public static void stringToFile(String source, File file) {
 	 */
 	public static boolean listToFile(List<String> source, String targetLocation) {
 
-		try {
-
-			File file = new File(targetLocation);
-			OutputStreamWriter out = new OutputStreamWriter(new FileOutputStream(file), StandardCharsets.UTF_8);
-
+		File file = new File(targetLocation);
+		try(OutputStreamWriter osw = new OutputStreamWriter(new FileOutputStream(file), StandardCharsets.UTF_8);
+			BufferedWriter bufferedWriter = new BufferedWriter(osw)) {
+			
 			for (String aSource : source) {
-
-				out.write(aSource + System.getProperty("line.separator"));
-
+				bufferedWriter.write(aSource + System.getProperty("line.separator"));
 			}
 
-			out.close();
+			bufferedWriter.close();
+			osw.close();
```

Comment:
Redundant since you're using try-with-resources anyway and they will be closed on scope exit.